tools/targets/__init__.py

```diff
@@ -675,6 +675,24 @@ def complete(t_self, resources, elf, binf):
         else:
             psoc6_complete(t_self, elf, binf)
 
+    @staticmethod
+    def sign_image(t_self, resources, elf, binf):
+        """
+        Calls sign_image function to add signature to Secure Boot binary file.
+        """
+        version = sys.version_info
+
+        # check python version before calling post build as is supports only python3+
+        if((version[0] < 3) is True):
+            t_self.notify.info("[PSOC6.sing_image] Be careful - produced HEX file was not signed and thus "
+                               "is not compatible with Cypress Secure Boot target. "
+                               "You are using Python " + str(sys.version[:5]) + 
+                               " which is not supported by CySecureTools. "
+                               "Consider installing Python 3.7+ and rebuild target. "
```

Comment:
Should this be now 3+ instead of 3.7+. As discussed earlier that Mbed OS repository part works almost any python 3 version

Reply:
Well basically it works with 3.4+, as one of the libraries we use (pathlib) was introduced from 3.4 - i made a change